hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/ClassAndFieldDefinitionTest.java

```diff
@@ -55,8 +56,7 @@ public void testClassDef_getField_properIndex() throws Exception {
     @Test
     public void testClassDef_hasField() throws Exception {
         for (int i = 0; i < classDefinition.getFieldCount(); i++) {
-            FieldDefinition field = classDefinition.getField(i);
-            assertNotNull(field);
+            assertTrue(classDefinition.hasField(fieldNames[i]));
```

Comment:
I would be careful with such 'complex' expression in an assertTrue. Because it isn't immediately clear what is being asserted here (it doesn't read as a good book).